sklearn/mixture/gaussian_mixture.py

```diff
@@ -648,3 +672,22 @@ def aic(self, X):
             The greater the better.
         """
         return -2 * self.score(X) * X.shape[0] + 2 * self._n_parameters()
+
+    @property
+    def precisions_(self):
+        self._check_is_fitted()
+        if self.precision_type is 'full':
+            return np.array([np.dot(prec, prec.T)
+                            for prec in self._precisions])
+        elif self.precision_type is 'tied':
+            return np.dot(self._precisions, self._precisions.T)
+        else:
+            return self._precisions
+
+    @property
+    def precisions_cholesky_(self):
+        self._check_is_fitted()
+        if self.precision_type in ['full', 'tied']:
+            return self._precisions
+        else:
+            return np.sqrt(self._precisions)
```

Comment:
I think it would be nicer to drop the `_precisions` private attribute and use and store `precisions_cholesky_` directly (instead of using a property for that attribute).

We can them change the `precisions_` property to rely upon `precisions_cholesky_` instead of `_precisions`.

What do you think?